builder-20/eol-buildpack/bin/build

```
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+function display_error() {
+  local ansi_red="\033[1;31m"
+  local ansi_reset="\033[0m"
+  local message=$1
+  echo
+  while IFS= read -r line; do
+    echo -e "${ansi_red}${line}${ansi_reset}" >&2
+  done <<< "$message"
+  echo
+}
+
+read -r -d '' EOL_MESSAGE <<'EOF' || true
+#######################################################################
+ _                    _               ___   ___    ______ ____  _
+| |                  | |             |__ \ / _ \  |  ____/ __ \| |
+| |__   ___ _ __ ___ | | ___   _ ______ ) | | | | | |__ | |  | | |
+| '_ \ / _ \ '__/ _ \| |/ / | | |______/ /| | | | |  __|| |  | | |
+| | | |  __/ | | (_) |   <| |_| |     / /_| |_| | | |___| |__| | |____
+|_| |_|\___|_|  \___/|_|\_\\\__,_|    |____|\___/  |______\____/|______|
+
+This builder image ('heroku/bulder:20') is deprecated, since it is
+based on the deprecated 'heroku/heroku:20' base image.
+
+Starting April 30th, 2025, this image will no longer receive security
+updates. Shortly after, this builder will be disabled and made
+unavailable.
+
+To continue receiving security updates and avoid interruption, upgrade
+to one of our newer builders, such as 'heroku/builder:22' or
+'heroku/builder:24':
+https://github.com/heroku/cnb-builder-images#heroku-cnb-builder-images
```

(The readme headings have changed since #429)

```suggestion
https://github.com/heroku/cnb-builder-images#available-images
```